controlnext/templates/controlnext/basin_detail.html

```diff
@@ -132,6 +136,9 @@ <h5 data-bind="text: moment().format('D MMMM YYYY') + ' (week: ' +  currentweek
                     </div>
                 </div>
                 <div class="col-sm-3">
+                    <a class="btn btn-info btn-sm" role="button"
+                       style="margin-left:0px;margin-top:5px;width:160px;"

```

Comment:
Moet de style niet in css?

En anders: een spatie achter de puntkomma verbetert de leesbaarheid.